zmon_worker_monitor/builtins/plugins/memcached.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from pymemcache.client.base import Client
+
+from zmon_worker_monitor.zmon_worker.errors import ConfigurationError
+
+from zmon_worker_monitor.adapters.ifunctionfactory_plugin import IFunctionFactoryPlugin, propartial
+from zmon_worker_monitor import plugin_manager
+
+COUNTER_KEYS = frozenset([
```

Comment:
Do we need all those keys by default?

Reply:
probably not ... checking which ones should be in by default